api/data/filters.sql

```diff
@@ -1,38 +1,44 @@
 DELETE FROM public.page_filters;
-
-INSERT INTO public.page_filters VALUES ('sector', 'Agriculture;Forestry;Both');
+-- Sections in the survey
+-- Section 1 - General Information
+INSERT INTO public.page_filters VALUES ('organisation-by-sector', 'Agriculture;Forestry;Both');
```

Comment:
Given that we are now storing all the data in our side, at some point we will need to deprecate this approach, and get the data from the real data itself, selecting distinct values given something

This way we avoid maintaining 2 sources of truth that can potentially create inconsistencies, most likely from "the other side".